### PR TITLE
DOCS-1615: Fixing issues found in testing

### PR DESCRIPTION
**Pull request by girishshankaran**
I'm validating Calico install steps on a kind multi-node cluster.

Product Version(s):
Calico 3.24, 3.25, 3.26, and version next.

Issue:
https://tigera.atlassian.net/browse/DOCS-1615

Link to docs preview:

SME review:
- [ ] An SME has approved this change. 

DOCS review:
- [ ] A member of the docs team has approved this change.

Additional information:

Merge checklist:

- [ ] Deploy preview inspected wherever changes were made 
- [ ] Build completed successfully
- [ ] Test have passed

### CODE REVIEW

**Comment by girishshankaran** [comment]
@lwr20 Can you please review the change specific to?

kind create cluster --config `kind.yaml` --name dev
kind create cluster --config `values.yaml` --name dev

**Comment by lwr20** [comment]
> @lwr20 Can you please review the change specific to?
> 
> kind create cluster --config `kind.yaml` --name dev 
> kind create cluster --config `values.yaml` --name dev

That looks sane to me, since you wrote the `values.yaml` immediately above.  But you should run through the doc yourself on a linux machine and check it works :)

**Comment by girishshankaran** [comment]
> > @lwr20 Can you please review the change specific to?
> > kind create cluster --config `kind.yaml` --name dev
> > kind create cluster --config `values.yaml` --name dev
> 
> That looks sane to me, since you wrote the `values.yaml` immediately above. But you should run through the doc yourself on a linux machine and check it works :)

Sorry, I did not explain it correctly. I was testing [these steps](https://docs.tigera.io/calico/latest/getting-started/kubernetes/kind), and I found that the "kind create cluster --config `kind.yaml` --name dev" command is not working. The command that worked is "kind create cluster --config `values.yaml` --name dev". So I wanted to make that fix in the docs.

File: calico/getting-started/kubernetes/kind.mdx [hunk 1]
```diff
@@ -36,6 +36,8 @@ KIND ships with a simple networking implementation ("kindnetd") based around sta
 
 Kind cluster creation can be modified by [configuration file](https://kind.sigs.k8s.io/docs/user/quick-start/#creating-a-cluster). Here you are providing kind configuration in order to disable simple networking implementation (default CNI) and change the pod ip CIDR.
 
+1. Disable the default CNI in order to use Calico CNI.
```
Comment (ctauchen) [inline calico/getting-started/kubernetes/kind.mdx]:
```suggestion
1. Disable the default CNI in order to use Calico CNI by running the following command:
```

File: calico/getting-started/kubernetes/kind.mdx [hunk 2]
```diff
@@ -50,13 +52,13 @@ networking:
 EOF
 ```
 
-Start your Kind cluster with one control plane and two worker nodes using the following command.
+2. Start your Kind cluster with one control plane and two worker nodes using the following command.
```
Comment (ctauchen) [inline calico/getting-started/kubernetes/kind.mdx]:
```suggestion
2. Start your Kind cluster with one control plane and two worker nodes by running the following command:
```

File: calico/getting-started/kubernetes/kind.mdx [hunk 3]
```diff
@@ -50,13 +52,13 @@ networking:
...
 
 ```bash
-kind create cluster --config kind.yaml --name dev
+kind create cluster --config values.yaml --name dev
 ```
 
-Confirm that you now have three nodes in your cluster with the following command.
+3. Confirm that you now have three nodes in your cluster with the following command.
```
Comment (ctauchen) [inline calico/getting-started/kubernetes/kind.mdx]:
```suggestion
3. Confirm that you have three nodes in your cluster by running the following command:
```

**Review by ctauchen — APPROVED** [review]
LGTM, with suggestions (please apply to 3.26 too)

File: calico/getting-started/kubernetes/kind.mdx (hunk 3 above)
Comment (girishshankaran) [reply calico/getting-started/kubernetes/kind.mdx]: Done

File: calico/getting-started/kubernetes/kind.mdx (hunk 2 above)
Comment (girishshankaran) [reply calico/getting-started/kubernetes/kind.mdx]: Done

File: calico/getting-started/kubernetes/kind.mdx (hunk 1 above)
Comment (girishshankaran) [reply calico/getting-started/kubernetes/kind.mdx]: Done

**Pull request merged**